etc/login.defs

```diff
@@ -440,7 +440,7 @@ USERGROUPS_ENAB yes
 # This option is overridden with the -M or -m flags on the useradd(8)
 # command-line.
 #
-#CREATE_HOME     yes
+CREATE_HOME     yes
```

Comment:
I think this change is unrelated to the test framework and should be addressed in a separate PR.  Or are you using it for testing purposes?

Reply:
I'm using it for testing purposes.

One of the objectives of the framework is to be able to test the same functionality in all distributions. In order to do this, a number of things have to be in place, including configuration sharing. For this particular case, both Debian and Fedora at distribution level have this configuration enabled, but Alpine does not. I could have created a new `login.defs file` just for testing but I decided to reuse the existing one. This will give more value to this sample `login.defs file` file, as it will see how it works in conjunction with the code.